doc/user/bgp.rst

```diff
@@ -1668,6 +1671,16 @@ address-family:
 
    Disables import or export of routes betwen the current unicast VRF and VPN.
 
+.. index:: import vrf VRFNAME
+.. clicmd:: import vrf VRFNAME
+
+   Enables direct importation of VRF routes to another VRF.  The RD and RT
+   are auto derived and are not needed for VRF - VRF route leaking.
+
+.. index:: no import vrf VRFNAME
+.. clicmd:: no import vrf VRFNAME
+
+   Disables direct VRF to VRF route leaking.
```

Comment:
Proposed: language parallel to proposed text for 'import vrf VRFNAME' above

Reply:
`Disables automatic leaking from vrf VRFNAME to the current VRF using the VPN RIB as intermediary`